index.js

```diff
@@ -0,0 +1,94 @@
+// function takes in an integer and outputs a random integer between 0 & input
+const randomInt = (max) => {
+  return Math.round(Math.random() * (max - 1))
+}
+
+// function accepts a integer of the size of the gameboard's sides
+// it outputs a 2-dimensional array, initialized with all `0` values
+// `0` represents a dead cell, `1` represents an alive cell
+const createBoard = sideLength => {
+  const board = []
+  for (let i = 0; i < sideLength; i++) {
+    board[i] = []
+  }
+  board.forEach(row => {
+    for (let j = 0; j < sideLength; j++) {
+      row[j] = 0
+    }
+  })
+  return board
+}
+// initializes a gameboard of 25 cells - NOTE: this can be altered (by user?)
+let board = createBoard(5)
+const side = board.length
+
+// function takes in a non-zero number of cells you want to select
+// and turns that many, randomly selected indices in the sub`board` to `1`
+const selectCells = num => {
+  for (let i = 0; i <= num; i++) {
+    const row = randomInt(side)
+    const cell = randomInt(side)
+    board[row][cell] = 1
+  }
+}
+// initializes the board with a random number of random squares selected
+// NOTE: input could be changed by user instead
+selectCells(randomInt(side))
+
+// function takes in the row & cell placement of a square and returns number
+// of live neighbors
+const liveNeighbors = (r, c) => {
```

Comment:
Try to use more readable variable names, It is very difficult to know what r or c means

Reply:
ah yeah I was a little torn because I used `row` and `cell` in the `forEach` callbacks and didn't want to conflict with the different scopes - I can try and come up with something else though

Reply:
Here's my two-cents:  r and c are fine as they are.  We are working with a matrix/grid and you clearly stated these were rows and columns in the comments.  Single letter names aren't always a great idea, but r and c are quite intuitive here considering the given content.  It is nitpicking to criticize their use here.

Reply:
okay - thanks for the feedback, both of yall!  I think I will leave r & c for now but I'll be sure to keep in mind more readable names in the future!
Much appreciated :)